configure.ac

```diff
@@ -193,6 +192,7 @@ AC_DEFINE_UNQUOTED(PASSWD_PROGRAM, "$shadow_cv_passwd_dir/passwd",
 	[Path to passwd program.])
 
 dnl XXX - quick hack, should disappear before anyone notices :).
+dnl XXX - I just read the above message :).
```

Comment:
Oh, and BTW, I sent this just for fun :)

Reply:
I thought you added that to retrigger the CI build. What do you plan to do?

Reply:
Nah, I was just joking about the message above message having been written with the intention that it would be removed quickly, and I noticed it 15 years later :)

```sh
$ grep -rl 'quick hack' | xargs git blame -- | grep 'quick hack'
45c6603cc configure.in (nekral-guest       2007-10-07 11:44:02 +0000 194) dnl XXX - quick hack, should disappear before anyone notices :).
```

And the commit message didn't explain what it did either:

```
$ git log -1 45c6603cc
commit 45c6603cc86c5881b00ac40e0f9fe548c30ff6be
Author: nekral-guest <nekral-guest@5a98b0ae-9ef6-0310-add3-de5d479b70d7>
Date:   Sun Oct 7 11:44:02 2007 +0000

    [svn-upgrade] Integrating new upstream version, shadow (19990709)
```

My intention is to remove `USE_SYSLOG`, but I'll do it in a different PR.  You can drop the `Ping? :)` commit from this PR when applying, or keep it just for fun (and as a reminder that we need to fix that); whatever you prefer :)

Reply:
For me, this PR is ready to merge; I don't expect to add more patches or change the existing ones (unless you find another rabbit hole where we can chop more code).